src/Layouts/Partials/AlignItemsPanel.vala

```diff
@@ -54,14 +68,10 @@ public class Akira.Layouts.Partials.AlignItemsPanel : Gtk.Grid {
         halign = Gtk.Align.FILL;
         get_style_context ().add_class ("alignment-box");
```

Comment:
Add the `column_homogeneous = true;` here.
We want the grid columns to expand, not the buttons, otherwise clicking on a white area will trigger a button and we don't want that.

Reply:
Ok, thanks for the explanations!